corehq/toggles.py

```diff
@@ -730,11 +730,13 @@ def _ensure_valid_randomness(randomness):
 )
 
 
-ROLE_WEBAPPS_PERMISSIONS = StaticToggle(
-    'role_webapps_permissions',
-    'ICDS: Toggle which webapps to see based on role',
+APP_ACCESS_PERMISSIONS = StaticToggle(
+    'app_access_permissions',
```

Comment:
Renaming this will change the id of the couch doc, so the domains currently using this flag will lose access. [Here's](https://www.commcarehq.org/hq/admin/raw_doc/?id=hqFeatureToggle-role_webapps_permissions) the current production doc for this flag. A django migration with a `RunPython` could be a good way to make sure everyone gets migrated on all servers.

Reply:
Yes, django migration sounds like the right way through it.
Also i think this cleanup (rename + migration) can be a follow up and need not necessarily block this PR, if you want to roll this out considering this was released and then had to be rolled back.

Reply:
To tackle this, I had manually listed down all the domains which were using this feature flag. I was planning to add them back when this is rolled out.
Running migration is definitely a better way to go about it.